pkg/services/ngalert/store/alert_rule.go

```diff
@@ -68,6 +68,13 @@ func (st DBstore) DeleteAlertRulesByUID(ctx context.Context, orgID int64, ruleUI
 			return err
 		}
 		logger.Debug("Deleted alert instances", "count", rows)
+
+		rows, err = sess.Table("alert_rule_state").Where("org_id = ?", orgID).In("rule_uid", ruleUID).Delete(alertRule{})
```

Comment:
> Delete(alertRule{})

There is no specific model for the data in `alert_rule_state`, so I just passed `alertRule` as in the query above for the `alert_instance`. The query doesn't return anything, so I think it's fine